HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -52,6 +52,31 @@ def customizeHLTfor47378(process):
     
     return process
 
+def customiseFor47926(process):
+
+    # if any of the following objects does not exist, do not apply any customisation
+    for objLabel in [
+        'hltSiStripRawToClustersFacility',
+        'HLTDoLocalStripSequence',
+        'HLTIterativeTrackingIteration0',
+        'hltIter0PFlowCkfTrackCandidates',
+    ]:
+        if not hasattr(process, objLabel):
+            print(f'# WARNING: customizeHLTIter0ToMkFit failed (object with label "{objLabel}" not found) - no customisation applied !')
+            return process
```

Comment:
To improve the "safety" of this customisation, why not check for the existence of the module `hltIter0PFlowCkfTrackCandidatesMkFitSiStripHits` ?

The deletion of `hltSiStripRecHits` should also probably be done only if that modules exists in the configuration.